docs/data/charts/scatter/scatter.md

```diff
@@ -109,7 +109,13 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 
 ### Shape 🚧
 
-### Size 🚧
+### Size
+
+You can customize the size of points in a scatter chart using the `markerSize` prop of every series.
+
+The `markerSize` is the radius of the point in pixels.
```

Comment:
```suggestion
You can customize the size of points in a scatter chart using the `markerSize` prop of every series.
For circles, the `markerSize` is the radius of the point in pixels.
```

Reply:
> For circles

We don't have any option at the moment, do we? 😅